src/Picker/SelectButton.js

```diff
@@ -180,15 +180,21 @@ SelectButton.propTypes = {
   /**
    * Optional children can be used to define content
    */
-  children: PropTypes.node
+  children: PropTypes.node,
+
+  /**
+   * Flag that allows the SelectButton to select same key and it will call the callback function
```

Comment:
I don't think that "allowSelectedSameKey" makes sense to the user of this component.

I am assuming what you mean by "Same key" is the currently selected key right?

Is there ever a case when we would not want to allow it?